.github/workflows/docs-check.yml

```diff
@@ -0,0 +1,20 @@
+name: Docs Check
+
+on:
+  pull_request:
+    branches:
+      - master
+    types:
+      - opened
+      - labeled
+      - unlabeled
```

Comment:
unlabeled is unnecessary, since we are tracking a label.

```suggestion
on:
  pull_request:
    types: [opened, labeled]
```

Reply:
I think the CI check should still run on an unlabeled PR - worst case it's just seen as one more passing test, but best case it might remind the submitter to add a docs label if it's relevant to do so. We also want it to re-run if the `docs-needed` label gets removed (e.g. if it was put there accidentally), which I'm assuming this would help with?